server/src/handlers/http/about.rs

```diff
@@ -44,9 +44,11 @@ pub async fn about() -> Json<serde_json::Value> {
     let store = CONFIG.storage().get_endpoint();
     let is_llm_active = &CONFIG.parseable.open_ai_key.is_some();
     let llm_provider = is_llm_active.then_some("OpenAI");
+    let ui_version = option_env!("UI_VERSION");
 
     Json(json!({
         "version": current_version,
+        "uiVersion": ui_version,
```

Comment:
if `UI_VERSION` is not set, let us send `Development Version` as the response in the API.